enterprise/models.py

```diff
@@ -1021,7 +1021,8 @@ def enroll(self, course_run_id, mode, cohort=None, source_slug=None, discount_pe
                 )
             )
             try:
-                enrollment_api_client.enroll_user_in_course(self.username, course_run_id, mode, cohort=cohort)
+                enrollment_api_client.enroll_user_in_course(self.username, course_run_id, mode, cohort=cohort,
+                                                            enterprise_uuid=str(self.enterprise_customer.uuid))
```

Comment:
[style nitpick] Could you move the first line of args onto their own line so the indentation adheres to our style guidelines?   Something like this:
```python
enrollment_api_client.enroll_user_in_course(
    self.username, 
    course_run_id, 
    mode, 
    cohort=cohort,
    enterprise_uuid=str(self.enterprise_customer.uuid),
)
```
https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#breaking-long-lines

Reply:
@iloveagent57  It's done